docs/graphql/core/migrations/advanced/resetting-migrations.rst

```diff
@@ -0,0 +1,63 @@
+.. meta::
+   :description: Resetting Hasura migrations
+   :keywords: hasura, docs, migration, reset migrations, clear migrations
+
+.. _reset_migration:
+
+Reset Hasura migrations
+==============================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+If you have a lot of migrations and you need to squash them or your current migration state on the local machine is corrupted, you can reset the state and create new migrations from the state that is on the server.
+
+Step 1: Delete the local migrations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Let's use top level titles (with this type of underline: `------` as explained in the [docs wiki](https://github.com/hasura/graphql-engine/wiki/How-to-write-docs-pages).

Reply:
Thanks for calling my attention to it.